File: back/src/main/java/org/zerock/mallapi/domain/OrderStatus.java
package org.zerock.mallapi.domain;

public enum OrderStatus {
    ORDER_CHECKING, //
    PAYMENT_CONFIRMED, //결제 완료
    PENDING,       // 결제 대기 중
    COMPLETED,     // 결제 완료
    CANCELLED,     // 주문 취소
    SHIPPED,       // 배송 중
    DELIVERED,     // 배송 완료
    RETURNED,      // 반품 완료
    REFUNDED;      // 환불 완료
}

File: client/src/types/orderStatus.ts
export enum OrderStatus {
    ORDER_CHECKING = "주문확인", //주문 확인
    PAYMENT_CONFIRMED = "결제완료", //지불 완료
    PENDING = "보류",
    COMPLETED = "완료",
    CANCELLED = "취소",
    SHIPPED = "배송중",
    DELIVERED = "배송완료",
    RETURNED = "회수됨",
    REFUNDED = "환불됨"
}